src/content/blog/you-probably-don't-need-eslint-config-prettier-or-eslint-plugin-prettier/index.mdx

```diff
@@ -0,0 +1,212 @@
+
+The following two tools have historically been used to help ESLint interact nicely with Prettier:
+
+- [`eslint-config-prettier`](https://github.com/prettier/eslint-config-prettier): An ESLint _shareable config_ that disables formatting-related rules
+- [`eslint-plugin-prettier`](https://github.com/prettier/eslint-plugin-prettier): An ESLint _plugin_ that runs Prettier as a rule inside ESLint
+
+I believe neither tool is useful in most projects anymore.
+This blog post will explain the purpose of each of them, their differences, and why I generally don't use either.
+
+## Recap: ESLint Customizations
+
+[ESLint](https://eslint.org) works by letting individually configure "rules", or checks on your codebase.
+ESLint will do the work of parsing your code into a format rules understand, passing the code to those rules, and letting you know about any reports emitted by those rules.
+
+ESLint is highly extensible: meaning you can customize many aspects of how it runs.
+The most common ways to customize it are:
+
+- **[Parsers](https://eslint.org/docs/latest/extend/ways-to-extend#custom-parsers)**: Replace ESLint's built-in JavaScript parser to support reading code with different syntax than native JavaScript
+- **[Plugins](https://eslint.org/docs/latest/extend/ways-to-extend#plugins)**: Provide a set of rules that are added to the list of rules available to be configured
+- **[Shareable Configs](https://eslint.org/docs/latest/extend/ways-to-extend#shareable-configs)**: Provide configuration options for any number of rules - either as a standalone npm package or from part of a _plugin_
+
+Note that _plugins_ and _shareable configs_ are two different things.
+
+- _**Plugins**_ make rules available without configuring those rules.
+- _**Shareable configs**_ configure rules that come with ESLint and/or were previously made available in plugins.
+
+### ESLint Customization Example: TypeScript
+
+If you use ESLint to lint TypeScript code then you're hopefully using all three customizations:
+
+- **Parser**: [`@typescript-eslint/parser`](https://typescript-eslint.io/packages/parser) to support parsing TypeScript code
+- **Plugin**: [`@typescript-eslint/eslint-plugin`](https://typescript-eslint.io/packages/eslint-plugin) to load in rules specific to TypeScript
+- **Shared Config(s)**: Shared settings packaged with `@typescript-eslint/eslint-plugin` that configure many rules at once, such as [`plugin:@typescript-eslint/recommended`](https://typescript-eslint.io/linting/configs#recommended).
+
+Note that the typescript-eslint _shareable configs_ come from the `@typescript-eslint/eslint-plugin` npm package.
+Hence the `plugin:` prefix in front of them: that's how ESLint knows where to find the configs.
+
+## `eslint-config-prettier`
```

Comment:
[Todo] I should reach out to the maintainers of this package to make sure I'm not misrepresenting it or otherwise being rude.

Reply:
I've reached out. Will keep this open as a reminder that I should give them each a business week to get back to me.